src/libraries/System.Net.Sockets/src/System/Net/Sockets/UnixDomainSocketEndPoint.cs

```diff
@@ -10,13 +11,24 @@ namespace System.Net.Sockets
     /// <summary>Represents a Unix Domain Socket endpoint as a path.</summary>
     public sealed partial class UnixDomainSocketEndPoint : EndPoint
     {
+        // taken from System.IO.PathInternal
+        private static readonly StringComparison s_filePathComparison = OperatingSystem.IsWindows() ||
+                                                    OperatingSystem.IsMacOS() ||
+                                                    OperatingSystem.IsIOS() ||
+                                                    OperatingSystem.IsTvOS() ||
+                                                    OperatingSystem.IsWatchOS()
+                                                    ? StringComparison.OrdinalIgnoreCase
+                                                    : StringComparison.Ordinal;
+
         private const AddressFamily EndPointAddressFamily = AddressFamily.Unix;
 
         private readonly string _path;
         private readonly byte[] _encodedPath;
+        private readonly bool _bounded;
```

Comment:
I assume you meant past form of "bind"
```suggestion
        private readonly bool _isBound;
```

Reply:
Typo is fixed. However, the flag is needed. `_fullPath_` is reused for two purposes:
1. Cached full path to the file if the path is not abstract
2. Full path to the file if `UnixDomainSocketEndPoint` represents bounded form (see `CreateUnboundEndPoint` method).

`_fullPath` is cached at the first call of `Equals` or `GetHashCode`. But this fact doesn't indicate that the Endpoint is bounded to the file.

Reply:
> Typo is fixed

Please put back the underscore prefix.

It would be good to explain the dichotomy on `_fullPath` in comments then. That is something that is hard to spot from the implementation.

Reply:
Omg, I accidentally missed it.